drivers/flash/Kconfig.nor

```diff
@@ -75,4 +75,13 @@ config SPI_NOR_IDLE_IN_DPD
 	  long periods, and when used the impact of waiting for mode
 	  enter and exit delays is acceptable.
 
+config SPI_NOR_HIGH_PERF_MODE
+	bool "Enable High Performance Mode"
```

Comment:
There is a compliance check warning stating:

> Boolean option 'SPI_NOR_HIGH_PERF_MODE' prompt must not start with 'Enable...'. Please
check Kconfig guidelines.

Suggestion: "Use High Performance Mode"